sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -458,6 +458,7 @@ private void removeAllStoredErrorLogFiles(UUID id) {
     private void removeStoredThrowable(UUID id) {
         mErrorReportCache.remove(id);
         ErrorLogHelper.removeStoredThrowableFile(id);
+        WrapperSdkExceptionManager.deleteWrapperExceptionData(id);
```

Comment:
Not your code change but we have to make sure that id is non-null in this method. Can you add `@NonNull` annotation for the parameter, add null checking for the id in the above method, `removeAllStoredErrorLogFiles`.